README.md

```diff
@@ -107,6 +107,29 @@ async function doSomething() {
 }
 ```
 
+#### Example with temporary refresh
+
+```javascript
+async function doSomething() {
+  const mutex = new Mutex(redisClient, 'lockingResource', {
+    lockTimeout: 120000,
+    refreshInterval: 15000
+  })
+  const lockAcquired = await mutex.tryAcquire()
+  if (!lockAcquired) {
+    return
+  }
+  try {
+    while (mutex.isAcquired) {
```

Comment:
`onLockLost` was not provided, so check is lock still acquire is unnecessary (unhandled rejection will happen)

Reply:
removed the check, is it correct now?